WasatchNET/Spectrometer.cs

```diff
@@ -2209,7 +2250,7 @@ public virtual double[] getSpectrum(bool forceNew=false)
                         sum[px] -= dark_[px];
 
                 // this should be enough to update the cached value
-                if (readTemperatureAfterSpectrum)
+                if (readTemperatureAfterSpectrum && eeprom.hasCooling)

```

Comment:
should be redundant now that we have lower level guard

Reply:
I was upset enough that redundant felt good...